scripts/qa_crowbarsetup.sh

```diff
@@ -492,8 +492,8 @@ function onadmin_prepare_sles_repos()
             add_mount "" "clouddata.cloud.suse.de:/srv/nfs/suse-$suseversion/install" "${targetdir_install}"
         fi
 
-        local REPO
-        for REPO in $slesrepolist ; do
+        local repo
+        for repo in $slesrepolist ; do
             local zypprepo=""
             [ "$WITHSLEUPDATES" != "" ] && zypprepo="$REPO"
```

Comment:
Whoops, looks like something broke during rebasing and resolving merge conflicts.  Will fix.